assets/js/components/Popup.js

```diff
@@ -74,6 +74,7 @@ module.exports = class Popup {
 
     const closestPopup = utils.closest(e.target, '.popup');
     if (closestPopup && closestPopup === this.popupHitBox) {
+      this.popupHitBox.setAttribute('aria-hidden', true);
```

Comment:
When the user clicks on the currently displayed popup, line 77 has the effect of setting `aria-hidden="true" on it, but doing nothing else. Is this the desired behaviour?